model/resource/container.yaml

```diff
@@ -82,3 +82,8 @@ groups:
         brief: >
           All the command arguments (including the command/executable itself) run by the container. [2]
         examples: [ 'otelcontribcol, --config, config.yaml' ]
+      - id: labels.<key>
```

Comment:
Thanks for pointing this out @AlexanderWert! I'm not sure if there is any specific timeline for the releases in the build-tools repo (most probably not). So maybe @arminru you could help here since I see you are cutting the [releases](https://github.com/open-telemetry/build-tools/releases) in that project?

If we cut a new release I can send a PR to update the dependency in this current repo.

Reply:
We released it today: https://github.com/open-telemetry/build-tools/releases/tag/v0.21.0.

Reply:
Opened https://github.com/open-telemetry/semantic-conventions/pull/310 to bump up the version.

Reply:
Thanks @ChrsMark! I just merged it.